### DISCUSSION TITLE
App beschreiben...

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#36] [Q&A]
Ich bin im AppBuilder auf der Suche nach "App beschreiben..." gleich wie im MapBuilder "Map beschreiben...", die Möglichkeit scheint es aber nicht zu geben?

**Comment by gis-eni** [comment]
Stimmt diese Möglichkeit gibt es nicht.
Zur Zeit ist auch nicht angedacht, dass das umgesetzt wird. Die Beschreibung wird in Viewer unter "Info" angezeigt. Das gibt es in den Apps allerdings nicht. Darum würde die Beschreibung "nur" in der Kachel auf der Startseite stehen, wo wenig Patz ist.

Wenn man eine Beschreibung in der App möchte, muss man in App Template irgendwo Platz dafür schaffen. Dann könnte man das Template auch so umschreiben, dass man die Beschreibung über einen Eingabefeld im AppBuilder festlegen kann (genau so, wie alle anderen Parameter)

**Comment by gis-eni** [comment]
Wenn es bei der Beschreibung nur darum geht, dass in der Kachel auf der Startseite ein Zusätzlicher Text angezeigt wird, ist das mit Build 5.23.901 umgesetzt
